buffer/src/main/java/io/netty/buffer/PooledSlicedByteBuf.java

```diff
@@ -340,14 +340,14 @@ protected void _setLong(int index, long value) {
 
     @Override
     public ByteBuf setLongLE(int index, long value) {
-        checkIndex0(index, 8);
-        unwrap().setLongLE(idx(index), value);
+        checkIndex(index, 8);
+        _setLongLE(index, value);
         return this;
     }
 
     @Override
     protected void _setLongLE(int index, long value) {
-        unwrap().setLongLE(idx(index), value);
+        unwrap()._setLongLE(idx(index), value);
```

Comment:
@normanmaurer I have found this bug too, was really unnoticeable...

Reply:
@franz1981 not sure if we should do this as a separate PR ? Maybe overkill tho